### ISSUE TITLE
Mumble win32 does not build in debug mode

### ISSUE THREAD

**Issue by NoAglets** [mumble-voip/mumble#2062]
While I have been able to build the debug mumble dependencies, I have not been able to build the debug version of mumble using the following steps:
1) Click the "MumbleBuild - cmd" shortcut to launch a Windows command prompt.
2) git clone --recursive https://github.com/mumble-voip/mumble.git mumble
3) cd mumble
4) qmake -recursive main.pro CONFIG+="debug static no-elevation no-g15 no-asio" CONFIG-=sse2
5) nmake

**Comment by mkrautz** [comment]
Just for clarification, I assume this is 1.2.x since you're using CONFIG-=sse2?

**Comment by NoAglets** [comment]
I just figured out a way to build the debug version.
replace: qmake -recursive main.pro CONFIG+="release static no-elevation no-g15 no-asio" CONFIG-=sse2
with: qmake -recursive main.pro CONFIG+="debug static no-elevation no-g15 no-asio" CONFIG-=sse2 DEFINES+=_SCL_SECURE_NO_WARNINGS

mkrautz:
Thanks for your reply. Actually, I was building the 1.3.x. I was not aware of the meaning of the sse2 CONFIG value. I started from scratch and used this time I used:
qmake -recursive main.pro CONFIG+="debug static no-elevation no-g15 no-asio" CONFIG-=sse3 DEFINES+=_SCL_SECURE_NO_WARNINGS

It also built fine as well.
